### ISSUE TITLE
rule: container-no-align-content

### ISSUE THREAD

**Issue by purupurupu** [purupurupu/css-noop-checker#125]
## Rule ID
`container-no-align-content`

## What does this rule detect?
`align-content` has no effect on single-line flex containers (`flex-wrap: nowrap`). The Flexbox spec says single-line containers stretch their one flex line and ignore `align-content`.

## Example: should warn
```html
<div style="display: flex; flex-wrap: nowrap; align-content: center;">
  <span>item</span>
</div>
```

## Example: should NOT warn
```html
<div style="display: flex; flex-wrap: wrap; align-content: center;">
  <span>item</span>
</div>
```

## Related CSS Properties
`display`, `flex-wrap`, `align-content`

## References
- [CSS Flexbox Level 1 §8.4 - align-content](https://www.w3.org/TR/css-flexbox-1/#align-content-property): "has no effect on a single-line flex container"
- One of the most common flexbox mistakes
